src/main/java/com/agilefaq/codedoctorsample/App.java

```diff
@@ -22,6 +22,37 @@ public static int multiply(int a1, int a2) {
         if (point1 == point) {
             System.out.println("Equal");
         }
+
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        if (a1 == 1) {
+            a1 = 2;
+        }
+        return a1 * a2;
+    }
+
+    public static int multiply99(int a1, int a2) {
+        Point point = new Point();
+        Point point1 = new Point();
+        if (point1 == point) {
+            System.out.println("Equal");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3AS106)